dpnp/dpnp_iface_counting.py

```diff
@@ -72,6 +75,8 @@ def count_nonzero(x1, axis=None, *, keepdims=False):
     5
 
     """
+    if config.__DPNP_OUTPUT_DPCTL__:
+        return call_origin(numpy.count_nonzero, x1, axis, keepdims=keepdims)
```

Comment:
We made changes in count_nonzero func implementation. Is this fallback still required?

Reply:
I don't think so.